xbmc/windowing/X11/XRandR.h

```diff
@@ -39,23 +39,26 @@ class XMode
       hz=0.0f;
       isPreferred=false;
       isCurrent=false;
+      isInterlaced=false;
       w=h=0;
     }
   bool operator==(XMode& mode) const
     {
-      if (id!=mode.id)
+      if (id != mode.id)
         return false;
-      if (name!=mode.name)
+      if (name != mode.name)
         return false;
-      if (hz!=mode.hz)
+      if (hz != mode.hz)
         return false;
-      if (isPreferred!=mode.isPreferred)
+      if (isPreferred != mode.isPreferred)
         return false;
-      if (isCurrent!=mode.isCurrent)
+      if (isCurrent != mode.isCurrent)
         return false;
-      if (w!=mode.w)
+      if (w != mode.w)
         return false;
-      if (h!=mode.h)
+      if (h != mode.h)
+        return false;
+      if (isInterlaced != mode.isInterlaced)
```

Comment:
you only can get here in case of error. names can't be identical with different interlace modes.